dask/dataframe/core.py

```diff
@@ -2811,7 +2811,14 @@ def nunique(self, split_every=None):
         return self.drop_duplicates(split_every=split_every).count()
 
     @derived_from(pd.Series)
-    def value_counts(self, split_every=None, split_out=1):
+    def value_counts(
+        self, split_every=None, split_out=1, sort=None, ascending=False, dropna=None
+    ):
+        if dropna is not None and PANDAS_VERSION < "1.1.0":
```

Comment:
Can you define a `PANDAS_GT_110` in `dd._compat`? Makes it easier to clean up in the future.

Reply:
:+1: